src/content/1.7/development/webservice/cheat-sheet.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cheat sheet
+weight: 2
+---
+
+# Cheat sheet for Webservice
+
+All these options can be added to your queries as query parameters (either `GET` or `POST` depending on the method used in your request).
+
+## Generic Options
+
+| Key | Value | Description |
+|-----|-------|-------------|
+| **output_format** | `XML, JSON` | Change the output format |
+| **ps_method** | `GET, POST, PUT, DELETE` | Override the HTTP method used for the request |
+
+## List options
+
+| Key | Value | Description |
+|-----|-------|-------------|
+| **display** | `[field1,field2 …]` | Only display fields in brackets |
+| **display** | `full` | Display all fields |
+
+| Key | Value | Description |
+|-----|-------|-------------|
```

Comment:
Do not duplicate the header, it's not needed. Otherwise add a section

Reply:
It's on purpose because each table is for a different parameter I'll try to add sub section with `###` but I don't the page to be too "heavy"

Reply:
It was not pretty with subsections, nor a simple text as title
So I ended up merging them